AK/DeprecatedFlyString.h

```diff
@@ -84,9 +84,9 @@ class DeprecatedFlyString {
     static void did_destroy_impl(Badge<StringImpl>, StringImpl&);
 
     template<typename... Ts>
-    [[nodiscard]] ALWAYS_INLINE constexpr bool is_one_of(Ts... strings) const
+    [[nodiscard]] ALWAYS_INLINE constexpr bool is_one_of(Ts const&... strings) const
     {
-        return (... || this->operator==(forward<Ts>(strings)));
+        return (... || this->operator==(forward<Ts const&>(strings)));
```

Comment:
`forward<const&>` doesn't actually do anything. But in any case, taking arguments by `const&` now means you can remove the call to `forward` entirely:
```c++
template<typename... Ts>
[[nodiscard]] ALWAYS_INLINE constexpr bool is_one_of(Ts const&... strings) const
{
    return (... || this->operator==(strings));
}
```

Reply:
Mhh wouldn't `T&&` and then a forward be even better, as that would allow Stringview to be passed by value, while strings would be passed by reference?

Reply:
Yep that would work too

Reply:
`Ts&&` is in the API for some of the other String functions, so I can do that here too.